website/utils/metaTagKey.ts

```diff
@@ -0,0 +1,3 @@
+export const metaTagKey = {
```

Comment:
I guess this could also be a default export

Reply:
👍